src/main/target/REVO/target.h

```diff
@@ -29,6 +29,9 @@
 #define TARGET_BOARD_IDENTIFIER "RVLT"
 #define USBD_PRODUCT_STRING     "Revolt"
 #define TARGET_DEFAULT_MIXER    MIXER_QUADX_1234
+#define USE_UART4
```

Comment:
Having this definition up here isn't ideal, since it makes the definition hard to read. Can you please move your definitions down to https://github.com/kmitchel/betaflight/blob/65fdd805e350cb4391af49a66a600cee321660ca/src/main/target/REVO/target.h#L198 ,

and guard them with a `#if defined(REVOLT)` and `#endif`?